packages/components/src/color-palette/index.js

```diff
@@ -55,7 +56,7 @@ export default function ColorPalette( {
 			<ColorPicker
 				color={ value }
 				onChangeComplete={ ( color ) => onChange( color.hex ) }
```

Comment:
As far as I know hex colors format don't include "alpha" information in it. This make me think this PR is not complete yet as there's no way for the users of this component to get the value of the selected alpha.

Should we switch to rgba if disableAlpha is false? should we do it every time? do we need a format prop? should we pass the whole color object. These are all options.

Reply:
Should this PR be reverted?

Reply:
ah... that might explain why it was disabled in the first place.
I think we should revert and then we can add to address other issues.
I prepped this PR for the revert: https://github.com/WordPress/gutenberg/pull/18220